self-hosting/README.md

```diff
@@ -0,0 +1,329 @@
+# HOSTING THE CHT APP
+
+The Community Health Toolkit (CHT) core framework has been packaged into a docker container to make it portable and easy to install. It is available from [dockerhub](https://hub.docker.com/r/medicmobile/medic-os). To learn more how to work with docker you could follow the tutorial [here](https://docker-curriculum.com/#getting-started) and the cheat sheet [here](https://www.docker.com/sites/default/files/d8/2019-09/docker-cheat-sheet.pdf).  
+
+## Installing Docker Prerequisites
+
+This Ideally depends on which operating system you are running. You need to first install these on your box before you can run the CHT docker containers.
+
+### Minimum Hardware/Software Requirements
```

Comment:
In the docs issue, Nick asked:

> Might want to add for self hosting that we know which versions are good to use. EX: Ubuntu 18.04, python2, etc...

I see this is still missing. Maybe also add OS requirements (eg Windows 10, MacOS 10.14, Ubuntu 18) and python 2. 

That said!! I ran the install with no issue on Ubuntu 20.04 but really didn't use it beyond just logging.  

Oh - update! Nick's concern about 20.04/python2 was around [prepare.sh](https://github.com/medic/cht-infrastructure/blob/master/self-hosting/prepare-system/ubuntu/prepare.sh). This script is out of scope I think for this self hosting guide? Either way, we should still add OS requirements.